templates/404.html

```diff
@@ -3,7 +3,7 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>(404) The page you were looking for doesn't exist.</title>
+  <title>(404) The page you were looking for doesn't exist.Try again</title>
```

Comment:
Could you please add the phrase as  "Please Try Again" ?

Reply:
Done.Also added malayalam version for the error message..

Reply:
Please update the PR